src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -58,7 +58,9 @@
 
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
                         kube_config=None, kube_context=None, no_wait=False, tags=None, distribution='auto', infrastructure='auto',
-                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="300"):
+                        auto_upgrade=None, cl_oid=None, onboarding_timeout="300", container_registry_repository=None, container_registry_username=None,
```

Comment:
If we are making this parameter change, then let's keep the default as "true" since for connect command, auto update of agents is enabled by default

Reply:
Also, have we tested normal onboarding(without pcr). Would be kind of good to validate non pcr scenarios too if not already checked

Reply:
> If we are making this parameter change, then let's keep the default as "true" since for connect command, auto update of agents is enabled by default

following the same implementation as update command.